Digitall.Warehouse.Domain/Entities/Products/Price.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Digitall.Warehouse.Domain.Entities.Products
+{
+    /// <summary>
+    /// Price complext type.
```

Comment:
Summary is used only here. Use it in every assembly or don't use at all.

Reply:
This is little different from the other entities - as you can see it it not exactly an Entitiy - it is a value type, which is represented as EF complex type. In this sense I thought the summary is need as information.